### PR TITLE
Add multi-package container for basic DESeq2 script

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

File: combinations/hash.tsv [hunk 1]
```diff
@@ -80,3 +80,4 @@ umi_tools=1.0.1,samtools=1.1.0
 icount=2.0.0,bedtools=2.28.0,pigz=2.3.4
 r-base=4.0.2,seacr=1.3,bedtools=2.29.2
 nanopolish=0.13.2,samtools=1.10,minimap2=2.17
+r-base=4.0.3,r-optparse=1.6.6,r-ggplot2=3.3.2,r-rcolorbrewer=1.1_2,r-pheatmap=1.0.12,bioconductor-deseq2=1.28.0,bioconductor-biocparallel=1.22.0	bioconda/extended-base-image:latest	0
```
Comment (bgruening) [inline combinations/hash.tsv:83]: is the extended image really needed?

Comment (drpatelh) [reply combinations/hash.tsv:83]: Ah, sorry. I don't know to be honest! I couldn't find any docs about which base images to specify. That image was being used for alot of the previous R-based multi-tool containers so I copied that across. Which one should I specify?

Comment (drpatelh) [reply combinations/hash.tsv:83]: Or is there a way I can test this myself @bgruening? I have very specific scripts (e.g. [here](https://github.com/drpatelh/rnaseq/blob/dsl2/bin/deseq2_qc.r)) that need to work with the mulled container so will know straight away whether its suited for purpose. If it helps I will need plotting functionality but not sure whether things like graphics devices etc are different between base images or whether the more basic linux utilities are suited for this purpose. Thanks!

**Comment by bgruening** [comment]
We should maybe also share the script ... Gaaxy is using a similar one.

**Comment by drpatelh** [comment]
Absolutely 👍 I wrote this version which has been tried and tested in other pipelines for a while now (e.g. nf-core/atacseq and nf-core/chipseq) but @macroscian (our in-house stats expert) will be writing a cooler version soon which is why I snuck in those additional packages in the last commit. Will share here when it has been added to the pipeline. Thanks again 🙂

**Comment by bgruening** [comment]
No problem! Thanks to you!

**Pull request merged**